packages/hooks/src/useIsSystemMaintenance.ts

```diff
@@ -0,0 +1,14 @@
+import { useStore } from '@deriv/stores';
+
+const useIsSystemMaintenance = () => {
+    const { client } = useStore();
+    const { account_status } = client;
+
+    const is_system_maintenance =
+        !!account_status?.cashier_validation &&
+        account_status.cashier_validation.some(validation => validation === 'system_maintenance');
```

Comment:
```suggestion
    const is_system_maintenance =
        account_status.cashier_validation?.some(validation => validation === 'system_maintenance') || false;
```

Reply:
As with the `useCashierLocked` I suggest we keep the initial logic from the methods that used to be in `GeneralStore`

Reply:
Though this version with checking `account_status.cashier_validation` and having a fallback `false` is good, so nvmd